### ISSUE TITLE
The maximum AUC

### ISSUE THREAD

**Issue by seharanul17** [Luoyonghong/Multivariate-Time-Series-Imputation-with-Generative-Adversarial-Networks#2]
I copied and executed this code, but the AUC does not reach 86%. The maximum AUC I got was about 83% though I run it several times. I wonder if the code is the paper's official code, and if there is any difference in hyperparameter settings or particular random seed.

**Comment by chooron** [comment]
I converted this code into tensorflow2.1, but when I run it, this code also cannot reach 86%, the maximum AUC I got was only 79%

**Comment by liguang-ops** [comment]
> I converted this code into tensorflow2.1, but when I run it, this code also cannot reach 86%, the maximum AUC I got was only 79%

could you share the code which is in the tensorflow2.1 version?  I need run the model on my private data, but i can't understand the code in tensorflow1.x version. it's too complex for me. thanks a lot.

**Comment by chooron** [comment]
> 
> 
> > I converted this code into tensorflow2.1, but when I run it, this code also cannot reach 86%, the maximum AUC I got was only 79%
> 
> could you share the code which is in the tensorflow2.1 version? I need run the model on my private data, but i can't understand the code in tensorflow1.x version. it's too complex for me. thanks a lot.
Actually I converted the code into the tf2.0 by using the script(TensorFlow 2.0 Upgrade Script) ,you can use it to realize the upgrade, and one thing you need notice is that the contrib module is replaced by other module in tf2, I hope you can convert it successfully

**Comment by liguang-ops** [comment]
@chooron thanks. i will have a try leter.
